resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/service/StreamingResourceServiceProxy.java

```diff
@@ -171,11 +171,11 @@ public Boolean addResource(final LeafResource leafResource) {
     public void initPostConstruct() {
         // Add resources to persistence
         LOGGER.info("Prepopulating using resource builder: {}", resourceBuilder);
-        resourceBuilder.get().stream()
-                .peek(entry -> LOGGER.debug(entry.toString()))
+        resourceBuilder.get()
                 .forEach(entry -> {
                     Resource rootResource = entry.getKey();
                     LeafResource leafResource = entry.getValue();
+                    LOGGER.info("Persistence add for {} -> {}", rootResource.getId(), leafResource.getId());

```

Comment:
only really want this to be logged at debug level in a production system

Reply:
In all other services this is the current behaviour, changed for consistency.
Could have all changed to INFO but as we are changing some caching things, i think it's useful to keep at info for a bit